mindsdb_sdk/agents.py

```diff
@@ -292,41 +303,33 @@ def completion_stream_v2(self, name, messages: List[dict]) -> Iterable[object]:
         return self.api.agent_completion_stream_v2(self.project.name, name, messages)
 
     def _create_default_knowledge_base(self, agent: Agent, name: str) -> KnowledgeBase:
-        # Make sure default ML engine for embeddings exists.
         try:
-            _ = self.ml_engines.get('langchain_embedding')
-        except AttributeError:
-            _ = self.ml_engines.create('langchain_embedding', 'langchain_embedding')
-        # Include API keys in embeddings.
-        if agent.provider == "mindsdb":
-            agent_model = self.models.get(agent.model_name)
-            training_options = json.loads(agent_model.data.get('training_options', '{}'))
-            training_options_using = training_options.get('using', {})
-            api_key_params = {k: v for k, v in training_options_using.items() if 'api_key' in k}
-            kb = self.knowledge_bases.create(name, params=api_key_params)
-        else:
+            # TODO: Use the agent's model credentials?
```

Comment:
yes, it is a question: how to set up embedding model parameters.
maybe to add option to update config (default embedding model via SDK?)

Reply:
Yes, that's a great idea, @ea-rus. Let me add that in.